cmd/kops/upgrade.go

```diff
@@ -23,7 +23,7 @@ import (
 // upgradeCmd represents the upgrade command
 var upgradeCmd = &cobra.Command{
 	Use:   "upgrade",
-	Short: "upgrade clusters",
+	Short: "Automates checking for and applying Kubernetes updates.",
```

Comment:
This is tricky. This upgrades a cluster to match the version in the stable channel. May result in a downgrade as well.

How do we say that in 4 words?